packages/ketcher-core/src/application/render/restruct/rergroup.js

```diff
@@ -63,19 +63,34 @@ class ReRGroup extends ReObject {
 
     if (ret) ret = ret.extend(BORDER_EXT, BORDER_EXT)
 
+    let attachmentPointsVBox = render.ctab.getAttachmentsPointsVBox(
+      this.getAtoms(render)
+    )
+    if (attachmentPointsVBox)
```

Comment:
Could you please add `{ }` for even single-line `if`?

Reply:
sure, will do